packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -296,32 +302,89 @@ function genCompatConfig(createVersion: string, loadVersion: string): CompatConf
 /**
  * Generates the cross-client compat config permutations.
  * This will resolve to one permutation where `CompatConfig.createVersion` is set to the current version and
- * `CompatConfig.loadVersion` is set to the delta (N-1) version. Then, a second permutation where `CompatConfig.createVersion`
- * is set to the delta (N-1) version and `CompatConfig.loadVersion` is set to the current version.
+ * `CompatConfig.loadVersion` is set to the delta version. Then, a second permutation where `CompatConfig.createVersion`
+ * is set to the delta version and `CompatConfig.loadVersion` is set to the current version.
+ * The delta versions will be:
+ * - N-1 and N-2, for "fast train" customers (i.e. \>=2.10.0 \<2.20.0, \>=2.20.0 \<2.30.0, etc.)
+ * - N-1 and N-2, for "slow train" customers (i.e. ^1.0.0, ^2.0.0, etc.)
+ * - LTS versions
  *
- * Note: `adjustMajorPublic` will be set to true when requesting versions. This will ensure that we test against
- * the latest **public** major release when using the N-1 version (instead of the most recent internal major release).
+ * @remarks
+ * Fast/slow trains refer to the different velocities that customers adopt new releases.
+ * Fast train customers integrate most minor releases quickly and saturate on a roughly 3-month
+ * cadence (this could be subject to change in the future).
+ * Slow train customers mainly integrate public major releases and may take much longer to saturate
+ * on any given release.
+ * We want to be able to test cross-client compat for both types of customers, so we generate permutations for
+ * N/N-1 and N/N-2 for both fast and slow trains.
  *
  * @internal
  */
 export const genCrossClientCompatConfig = (): CompatConfig[] => {
-	const currentVersion = getRequestedVersion(pkgVersion, 0);
+	const currentVersion = getRequestedVersion(
+		pkgVersion,
+		0,
+		false, // adjustMajorPublic = false
```

Comment:
The general syntax for this is the following:
```suggestion
		false, /* adjustMajorPublic */
```